docs/src/man/comparisons.md

```diff
@@ -235,6 +235,59 @@ The table below compares more advanced commands:
+|Rename column (in place) |`setnames(df, "x", "x_new")`|`rename!(df, :x => :x_new)`|
+|Rename multiple columns (in place) |`setnames(df, c("x", "y"), c("x_new", "y_new")`|`rename!(df, [:x, :y] => [:x_new, :y_new])`|
+|Pick columns            |`df[, list(x, y)]`          |`select(df, :x, :y)`|
+|Remove columns          |`df[, -"x" ]`               |`select(df, Not(:x))`|
+|Remove columns (in place) |`df[, c("x") := NULL ]`   |`select!(df, Not(:x))`|
+|Pick & transform columns|`df[, list(mean(x), y)]`    |`select(df, :x => mean, :y)`|
+|Pick rows               |`df[ x >= 1 ]`              |`filter(:x => >=(1), df)`|
+|Sort rows (in place)    |`setorder(df, x)`           |`sort!(df, :x)`|
+|Sort rows               |`df[ order(x) ]`            |`sort(df, :x)`|
+
+### Grouping data and aggregation
+
+|Operations              | data.table                                      | DataFrames.jl     |
+|:-----------------------|:------------------------------------------------|:------------------|
+|Reduce multiple values  |`df[, list(mean(x)), by = list(id) ]`            |`combine(groupby(df, :id), :x => mean)`|
+|Add new columns (in place) |`df[, x_mean := mean(x), by = list(id) ]`        |`transform!(groupby(df, :id), :x => mean)`|
+|Pick & transform columns|`df[, list(x_mean = mean(x), y), by = list(id) ]`|`select(groupby(df, :id), :x => mean, :y)`|
+
+### More advanced commands
+
+|Operations              | data.table                                      | DataFrames.jl             |
+|:-----------------------|:------------------------------------------------|:--------------------------|
+|Complex Function          |`df[, list(mean(x, na.rm = T)) ]`              |`combine(df, :x => x -> mean(skipmissing(x)))`|
+|Transform certain rows (in place) |`df[x<=0, x:=0 ]`                      |`transform!(df, :x => (x -> ifelse.(x .<= 0, 0, x)) => :x)`|
+|Transform several columns |`df[, list(max(x), min(y)) ]`                  |`combine(df, :x => maximum,  :y => minimum)`|
+|                          |`df[, lapply(.SD, mean), .SDcols = c("x", "y") ]`   |`combine(df, [:x, :y] .=> mean)`|
+|                          |`df[, lapply(.SD, mean), .SDcols = patterns("x*") ]`|`combine(df, names(df, r"^x") .=> mean)`|
+|                          |`df[, unlist(lapply(.SD, function(x) c(max=max(x), min=min(x)))), .SDcols = c("x", "y") ]`|`combine(df, ([:x, :y] .=> [maximum minimum])...)`|
+|Multivariate function     |`df[, list(cor(x,y)) ]`                        |`transform(df, [:x, :y] => cor)`|
+|Row-wise                  |`df[, min_xy := min(x, y), by = 1:nrow(df)]`   |`transform(df, [:x, :y] => ByRow(min))`|
+|                          |`df[, argmax_xy := which.max(.SD) , .SDcols = patterns("x*"), by = 1:nrow(df) ]`|`transform(df, AsTable(r"^x") => ByRow(argmax))`|
+|DataFrame as input        |`df[, head(.SD, 2) ]`                          |`combine(d -> first(d, 2), df)`|
```

Comment:
I am not sure what we want to show with this line.

Reply:
I lost track of my former self. I will remove for now. 
I think I wanted to show how DataFrames can easily manipulate the table itself like the idiomatic `.SD` of data.table. Since data.table people like me think this is one of the thing that makes data.table really nice and complete, I wanted to illustrate some sort of feature parity. This example is not that.